Updates to upgrade procedures (#65388)
openshift/openshift-docs: modules/updates-and-upgrades.adoc

Customers have no control over when a Central service update occurs. For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
Upgrades to the version of {rh-rhacscs-first} are considered part of the service update.

> Customers are responsible for timely {product-title-short} Secured Cluster services upgrades required to maintain compatibility with {product-title-managed-short}.

Reviewer: ```suggestion
Customers are responsible for timely {product-title-short} Secured Cluster services upgrades that are required to maintain compatibility with {product-title-managed-short}.
```
ISG specifies "that" for restrictive clauses.

> Red Hat recommends enabling automatic upgrades for Secured Clusters connected to  {product-title-managed-short}.

Reviewer: ```suggestion
Red Hat recommends enabling automatic upgrades for Secured Clusters that are connected to {product-title-managed-short}.
```
or
```suggestion
Red Hat recommends enabling automatic upgrades for Secured Clusters that connect to {product-title-managed-short}.
```

Author: I realize this is a weird distinction, but the clusters exist outside of RHACS and while they do connect to them, the component that manages the connection as a whole (Central) is within ACS so I think it's more clear to say "that are connected to", if that makes sense.

Upgrades to the version of {rh-rhacscs-first} are considered part of the service update.

The version of {product-title-managed-short} used on Secured Clusters must match the version of the Central instance of {product-title-managed-short} to ensure compatibility.
Customers are responsible for Secured Cluster services upgrades required to maintain this version compatibility.

> Red{&nbsp}Hat recommends maintaining secured cluster components at the same version as the Central it is connected to for optimal compatibility. The software version for Central is automatically updated. See the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix] for more information.

Reviewer 2: Since this module is part of CS only:
- There is **no** such thing as a **special Secured Cluster for CS**. It's always the same Secured Cluster _distribution_ for both CS and self-managed (the difference is _where_ one _connects to_: self-managed central or CS central).
- Only customers are responsible for Secured Cluster upgrades.
- At the same time we strongly recommend always upgrading Secured Cluster to the latest version shortly after the one gets released.
- Therefore we strongly recommend enabling automatic upgrades.

If I try to write it down, it'd look something like this instead of the current content.
```suggestion
Customers are responsible for timely {product-title-short} Secured Cluster services upgrades required to maintain compatibility with {product-title-managed-short}.
Red Hat recommends enabling automatic upgrades for Secured Clusters connected to  {product-title-managed-short}.
See the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix] for more information.
```
